app/src/main/java/com/duckduckgo/app/email/EmailJavascriptInterface.kt

```diff
@@ -46,6 +46,15 @@ class EmailJavascriptInterface(
         }
     }
 
+    @JavascriptInterface
+    fun getUserData(): String? {
```

Comment:
Signature doesn't need to be nullable
```suggestion
    fun getUserData(): String {
```